src/app/migrations/password/password.1.core.txt

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <form clrForm>
+      <cds-password labelLayout="hidden-label" control-width="shrink">
+        <label>input</label>
```

Comment:
Why the label say `input` and not `Password`

Reply:
I don't see what is bothering you here but if you want to change it I will.